portingdb/tables.py

```diff
@@ -227,16 +219,16 @@ class CollectionPackage(TableBase):
     nonblocking = Column(
         Boolean(), default=False,
         doc=u"If true, does not block dependent packages (even if it's marked as unported)")
+    is_misnamed = Column(
+        Boolean(), doc=u"True if the package does not follow the naming policy")
```

Comment:
I tough this is Python 3 only, but I see that other docs also have `u""`, so keep it to be consistent.